src/compiler/msvc.rs

```diff
@@ -505,9 +508,6 @@ where
         .env_clear()
         .envs(env_vars.iter().map(|&(ref k, ref v)| (k, v)))
         .current_dir(&cwd);
-    if parsed_args.depfile.is_some() || parsed_args.msvc_show_includes {
```

Comment:
I think you need to keep the `parsed_args.depfile.is_some()` conditional around. If somebody has passed `dep` and not/instead of `showIncludes`